nixos/modules/rename.nix

```diff
@@ -135,7 +135,7 @@ with lib;
     # piwik was renamed to matomo
     (mkRenamedOptionModule [ "services" "piwik" "enable" ] [ "services" "matomo" "enable" ])
     (mkRenamedOptionModule [ "services" "piwik" "webServerUser" ] [ "services" "matomo" "webServerUser" ])
-    (mkRenamedOptionModule [ "services" "piwik" "phpfpmProcessManagerConfig" ] [ "services" "matomo" "phpfpmProcessManagerConfig" ])
+    (mkRemovedOptionModule [ "services" "piwik" "phpfpmProcessManagerConfig" ] "Use services.phpfpm.pools.<name>.settings")
```

Comment:
Shouldn't there be two lines for both `[ "services" "piwik" "phpfpmProcessManagerConfig" ]` and `[ "services" "matomo" "phpfpmProcessManagerConfig" ]` so that people using the latter get more informative error message?

Reply:
Yeah that was the idea... hmm...